test/integration/ipamd/ipamd_suite_test.go

```diff
@@ -88,6 +91,12 @@ var _ = BeforeSuite(func() {
 	primaryInstance, err = f.CloudServices.EC2().DescribeInstance(instanceID)
 	Expect(err).ToNot(HaveOccurred())
 
+	primaryInstanceDefaults, err := f.CloudServices.EC2().DescribeInstanceType(*primaryInstance.InstanceType)
+	Expect(err).ToNot(HaveOccurred())
+
+	defaultEniCount = len(primaryInstanceDefaults[0].NetworkInfo.NetworkCards)
+	defaultIpsPerEni = int(*primaryInstanceDefaults[0].NetworkInfo.Ipv4AddressesPerInterface)
```

Comment:
Quick question: if test cases run before this (like the CNI suite) and have not cleaned up yet, will this default count be off?

Reply:
No, this is being pulled from `instance-type` metadata. So it should be consistent irrespective of what tests were run